orchestrator/careplanservice/handle_createpatient.go

```diff
@@ -0,0 +1,83 @@
+package careplanservice
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/SanteonNL/orca/orchestrator/lib/audit"
+	"github.com/SanteonNL/orca/orchestrator/lib/auth"
+	"github.com/SanteonNL/orca/orchestrator/lib/coolfhir"
+	"github.com/SanteonNL/orca/orchestrator/lib/to"
+	"github.com/google/uuid"
+	"github.com/rs/zerolog/log"
+	"github.com/zorgbijjou/golang-fhir-models/fhir-models/fhir"
+)
+
+func (s *Service) handleCreatePatient(ctx context.Context, request FHIRHandlerRequest, tx *coolfhir.BundleBuilder) (FHIRHandlerResult, error) {
+	log.Ctx(ctx).Info().Msg("Creating Patient")
+	var patient fhir.Patient
+	if err := json.Unmarshal(request.ResourceData, &patient); err != nil {
+		return nil, fmt.Errorf("invalid %T: %w", patient, coolfhir.BadRequestError(err))
+	}
+
+	// Check we're only allowing secure external literal references
+	if err := s.validateLiteralReferences(ctx, &patient); err != nil {
+		return nil, err
+	}
+
+	// Patient.id must be generated by the backing FHIR API, so it must be nil in the request
+	patient.Id = nil
+
+	// Get the current principal
+	principal, err := auth.PrincipalFromContext(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	// Get local identity for audit
+	localIdentity, err := s.getLocalIdentity()
+	if err != nil {
+		return nil, fmt.Errorf("failed to get local identity: %w", err)
+	}
```

Comment:
I believe we need these for most, if not all, operations. It might be an idea to put them into the FHIRHandlerRequest struct?